### PR TITLE
Add an integration test policy for AWS WG

### PR DESCRIPTION
**Pull request by willthames**
Document a policy for tests for AWS modules

### CODE REVIEW

File: group-aws/integration.md [hunk 1]
```diff
@@ -3,6 +3,19 @@
 There are very few integration tests for EC2. Those that
 exist are in test/integration/targets
 
+## Policy
+
+Following the 2.4 release, all modules marked `stableinterface`
+MUST have integration tests for new features. Upgrading to boto3
+shall be considered a feature request.
+
```
Comment (gundalow) [inline group-aws/integration.md]: SHOULD/MUST all new modules be written to use boto3?

Comment (thaumos) [reply group-aws/integration.md]: yes, all new modules are to be boto3.  existing modules are case by case

File: group-aws/integration.md [hunk 2]
```diff
@@ -11,24 +24,22 @@ exist are in test/integration/targets
...
 | rds_param_group      | [25345](https://github.com/ansible/ansible/pull/25345) |
 
 # State of the codebase
 
 Existing modules with tests:
 * `aws_api_gateway`
+* `aws_lambda`
 * `aws_s3`
 * `ec2_ami`
-* `ec2_elb_lb`
```
Comment (s-hertel) [inline group-aws/integration.md]: Since the naming was undone and now ec2_elb_lb and elb_class_lb both have tests, this line should be put back.

**Comment by s-hertel** [comment]
LGTM

File: group-aws/integration.md (hunk 1 above)
Comment (ryansb) [reply group-aws/integration.md]: Migrations are happening, but no set roadmap for doing so.

Soon :tm:

**Comment by ryansb** [comment]
shipit

File: group-aws/integration.md (hunk 2 above)
Comment (willthames) [reply group-aws/integration.md]: Thanks @s-hertel, this is fixed now

**Pull request merged**